src/notebook/manager.ts

```diff
@@ -297,6 +309,99 @@ class NotebookManager {
   }
 
   /**
+   * Select the next cell.
+   */
+  selectNext(): void {
```

Comment:
We shouldn't call this "select.." if we are deselecting cells. Maybe "activateNext" since it is changing the active cell?